mkdocs.yml

```diff
@@ -0,0 +1,13 @@
+site_name: Defra Software Development Standards
+site_url: https://defra.github.io/software-development-standards
+theme: 
+  name: material
+  logo: img/code-white.png
+  favicon: img/code-black.png
+  palette:
+    primary: green
+  font:
+    text: 'Arial'
+    code: 'Roboto Mono'
+repo_url: https://github.com/DEFRA/software-development-standards
+remote_branch: gh-pages
```

Comment:
This can be extended to support a tree of specific paths, but given the structure we've got, it seems alphabetic order is ok.  Not like we're telling a linked story at any point.

Also reduces having to maintain another list.